tests/flow/test_harmonic_centrality.py

```
@@ -0,0 +1,279 @@
+from common import *
```

Ruff (0.15.12)

[error] `from common import *` used; unable to detect undefined names

(F403)